forestserviceprototype/specialuseform/templates/specialuseform/submitted_permit.html

```diff
@@ -1,8 +1,30 @@
 {% extends "specialuseform/base.html" %}
 
 {% block content %}
-  <h1>Thank you!</h1>
-  <p class="usa-font-lead">
-    We've got it! We'll be in touch.
-  </p>
+  <h1>Recieved Application</h1>
```

Comment:
Changes made in #17 weren't reapplied here — can you check (or just git pull that branch and remerge it)?